### ISSUE TITLE
Can’t use vim when SSHing into remote

### ISSUE THREAD

**Issue by uajqq** [blinksh/blink#1093]
I'm just switching to Blink from my old iPad terminal app (it's a game changer, thank you), and for some reason whenever I ssh into my remote machine (a Mac), I can't use vim. I get the following error:

```
Fatal Python error: config_get_locale_encoding: failed to get the locale encoding: nl_langinfo(CODESET) failed
Python runtime state: preinitialized
```

nvim also throws weird errors about not being configured with the proper Python. This doesn't happen when I access the machine using any other terminal app and doesn't happen when I use mosh through Blink (I do have to manually specify the location of mosh-server to connect successfully, if that's relevant). Remote is on Python 3.8.3 via pyenv. I assume I'm missing something obvious but can't figure it out.

**Comment by yury** [comment]
Hi @uajqq 

What is output of `locale` command on your Mac?

**Comment by uajqq** [comment]
When run via mosh:
```
LANG=""
LC_COLLATE="en_US.UTF-8"
LC_CTYPE="en_US.UTF-8"
LC_MESSAGES="en_US.UTF-8"
LC_MONETARY="en_US.UTF-8"
LC_NUMERIC="en_US.UTF-8"
LC_TIME="en_US.UTF-8"
LC_ALL="en_US.UTF-8"
```

When run via SSH:
```
LANG=""
LC_COLLATE="C"
LC_CTYPE="C"
LC_MESSAGES="C"
LC_MONETARY="C"
LC_NUMERIC="C"
LC_TIME="C"
LC_ALL=
```

**Comment by uajqq** [comment]
Also get similar errors even in mosh when trying to save files:
```
Fatal Python error: config_get_locale_encoding: failed to get the locale encoding:
 nl_langinfo(CODESET) failed             
Python runtime state: preinitialized 
```

**Comment by redreceipt** [comment]
I’m seeing similar issues. Seems that Blink is not sending locale info over SSH (which actually causes other problems with mosh downstream). Created a similar issue that shows the behavior. #1171

**Comment by redreceipt** [comment]
On Mac the setting is `SendEnv` but not sure where to set or see the ssh client config on Blink

**Comment by yury** [comment]
@redreceipt we support that option.

`ssh -o "SendEnv=ENVNAME" host`

**Comment by redreceipt** [comment]
> @redreceipt we support that option.
> 
> 
> 
> `ssh -o "SendEnv=ENVNAME" host`

That's not what I mean. The default setting in `/etc/ssh/ssh_config` on a macOS is 

```
Host *
  SendEnv LC_C*
```

Without send locale environment variables over SSH, I think Mosh is having trouble finding `/usr/local/bin/mosh-server` even though it's in my path. I have to specify the location for Mosh to work.

**Comment by carloscabanero** [comment]
I think it is separate issues, not being found on the path on MacOS may not have anything to do with exporting the locale. You can see this issue ie https://github.com/mobile-shell/mosh/issues/457.

In any case, sending the environment is supported as well, as Yury mentioned.

**Comment by carloscabanero** [comment]
Fixed on 13.5.11. We do not have all the LC_C* variables, but exporting LANG has the same effect. There is additionally a .blink/profile file, as a key value for environment variables, that is applied on every session.
